deploy/sre-prometheus/100-node-unschedulable.PrometheusRule.yaml

```diff
@@ -18,4 +18,4 @@ spec:
         namespace: openshift-monitoring
       annotations:
         message: "The node {{ $labels.node }} has been unscedulable for more than an hour."
-        link: "https://github.com/openshift/ops-sop/blob/master/v3/alerts/openshift_node_not_schedulable.asciidoc"
+        link: "https://github.com/openshift/ops-sop/blob/master/v4/alerts/KubeNodeUnschedulableSRE.md"
```

Comment:
If we remove the link completely, this will revert to the default link
```suggestion
```

Reply:
Thanks Christoph! Added another commit to remove the URL